### DISCUSSION TITLE
Can I use marimo as Streamlit to write full-stack applications

### DISCUSSION THREAD

**Discussion by lacia-hIE** [marimo-team/marimo#3196] [Q&A] [answered]
Can I use marimo as Streamlit to write full-stack applications

**Comment by akshayka** [comment, marked as answer]
You can use marimo as an alternative to streamlit. You may find the performance in marimo is better as well.

Here is a comparison of marimo and streamlit: https://docs.marimo.io/guides/coming_from/streamlit.html
